### DISCUSSION TITLE
Changing Color  button in App Bar

### DISCUSSION THREAD

**Discussion by AGM-90** [mui/toolpad#4492] [Help and support]
I manged to change color of App Bar in toolpad
How to change color of menu button and theme switcher

**Comment by apedroferreira** [comment]
Hi, we've had a similar discussion in https://github.com/mui/toolpad/issues/4291

To change those colors, maybe you can use the `sx` prop in DashboardLayout and target the icon buttons in the header/AppBar?

Other than that we can consider adding a slot to be able to override the menu button if it's not customizable yet.

**Comment by prakhargupta1** [comment]
@AGM-90 Were you able to do it through `sx` prop in DashboardLayout?
